landing_page/app/views/users/show.html.erb

```diff
@@ -1,80 +1,24 @@
-<header class="navbar navbar-fixed-top navbar-inverse">
-
-  <div class="container">
-
-    <nav>
-
-      <ul class="nav navbar-nav navbar-right">
-
-        <% if logged_in? %>
-
-          <li class="postcreation ">
-
-            <%= link_to "Create Blog Post", new_micropost_url %>
-
-          </li>
-
-          <li class="dropdown">
-
-            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
-
-              Account <b class="caret"></b>
-
-            </a>
-
-            <ul class="dropdown-menu">
-
-              <li><%= link_to "Edit Profile", edit_user_url%></li>
-              <li><%= link_to "View Blog Posts", microposts_path%></li>
-
-              <li class="divider"></li>
-
-              <li>
-
-                <%= link_to "Log out", logout_path, method: "delete" %>
-
-              </li>
-
-            </ul>
-
-          </li>
-        
-        <div class="fixed-top">
-          <% if logged_in? %>
-            <strong>Welcome, </strong>
-            <b><%= @user.email%></b>
-          <% end %>
-
-        <% else %>
-
-          <li><%= link_to "Log in", login_path %></li>
-
-        <% end %>
-
-      </ul>
-
-    </nav>
-   
-    
-
-  </div>
-
-</header>
-
-<p id="notice"><%= notice %></p>
-
-<p>
-  <strong>First name:</strong>
-  <%= @user.first_name %>
-</p>
-
-<p>
-  <strong>Second name: </strong>
-  <%= @user.second_name %>
-</p>
-
-<p>
-  <strong>Email:</strong>
-  <%= @user.email %>
-</p>
-<br></br>
+<div class="navbar">
+  <strong><u><%= link_to "Create Blog Post", new_micropost_url %></u></strong>
+  <strong><u><%= link_to "View Blog Posts", microposts_path%></u></strong>
+  <strong><u><%= link_to "View Archived Posts", microposts_path(@micropost, :archived => 'true')%></u></strong>
+  <strong><u><%= link_to "Edit Profile", edit_user_url%></u></strong>
+  <strong><u><%= link_to "Log out", logout_path, method: "delete" %></u></strong>
+</div>
+
+<%= react_component("Users", {title: "Your Profile Details"}) %>
+
+<div class="user_info">
+  <%= react_component("Users", {userInfoType: "First Name: "})%>
```

Comment:
I would have done this differently - Consider creating a React component called UserDetalis and pass it all the user details required. The component would then render lines 9 - 21. 
In this way there is less duplication with the `<%= react_component("Users",` call. 
So rather than having your view repeatedly call and piece together these react components all that responsibility should be managed within the react component itself. 
In this way you will make the code more modular and the view easier to comprehend, read and also less verbose.